libs/ledger-live-common/src/exchange/swap/getExchangeRates.ts

```diff
@@ -139,8 +139,10 @@ const inferError = (
     }
 
     // For out of range errors we will have a min/max pairing
-    if (minAmountFrom) {
-      const isTooSmall = new BigNumber(apiAmount).lte(minAmountFrom);
+    if (minAmountFrom || maxAmountFrom) {
+      const isTooSmall = minAmountFrom
```

Comment:
Should this be renamed to something like `amountIsOutOfRange` since it's not just looking at the minValue @sarneijim ?

Reply:
New variable created: 'hasAmountLimit'